examples/hcp-ec2-demo/output.tf

```
@@ -20,5 +20,23 @@ output "hashicups_url" {
 }
 
 output "next_steps" {
-  value = "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token."
+  value = local.install_demo_app ? "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token." : null
+}
+
+output "howto_connect" {
+  value = <<EOF
+  "In order to get access to both nomad and consul from the command line run the following commands:
```

```suggestion
  To access Nomad and Consul from the command line run the following commands:
```

I don't think we'll need the starting double quote

---

Can we just delete this line? I don't think `Nomad` will be there if `!var.install_demo_app` and in that case it's just Consul which already has some documentation below

---

Good point, deleting the line